test/integration/simple_zonal_with_asm/controls/kubectl.rb

```diff
@@ -41,18 +41,5 @@
         },
       )
     end
-
-    describe "Mesh" do
```

Comment:
Please don't remove the tests. If this needs to be updated to work with newer ASM, please adapt it but we want to continue verifying ASM was actually installed.

Reply:
Yeah I'm going to flesh out the tests to check for CPR existence. This one had to be removed since we don't install gateways in the new module.

Reply:
Expanded the tests.